packages/vuepress-theme-odyssey/components/DocsCardGroup.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <Component
+    :is="tag"
+    :class="{
+      'docs-card--group': true,
+      'is-2col': variant === '2col'
```

Comment:
> Not possible to do this automatically based on number of children / content?
> 
> If not, could we maybe create a use-case based name here?
> 
> _Originally posted by @edburyenegren-okta in https://github.com/okta/odyssey/pull/714#discussion_r509603399_

I probably could, but wanted to keep it a little more flexible. I would be interested in finding a CSS solve though, that somehow still made the component breakdown automatically at smaller widths. Ill follow up with you on slack about this.